lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/AddressService.java

```diff
@@ -64,7 +64,7 @@ public AddressItem saveAddresses(UUID sessionId, List<CanonicalAddress> addresse
         return addressItem;
     }
 
-    public AddressItem getAddresses(UUID sessionId) {
+    public AddressItem getAddress(UUID sessionId) {
```

Comment:
While I agree, in premise, about renaming this - I think it should probably be getAddressItem, rather than getAddress, as it returns an Address Item, which contains the address(es) for that session.

Can we rename to getAddressItem instead?